src/txn/txn.c

```diff
@@ -2808,6 +2808,21 @@ __wt_verbose_dump_txn_one(
     txn = txn_session->txn;
     txn_shared = WT_SESSION_TXN_SHARED(txn_session);
 
+    if (txn->isolation != WT_ISO_READ_UNCOMMITTED &&
```

Comment:
It looks good. Some small suggestion:
```suggestion
    if (txn->isolation == WT_ISO_READ_UNCOMMITTED || F_ISSET(txn, WT_TXN_HAS_SNAPSHOT)) {
                 WT_RET(__wt_msg(session,
      "session ID: %" PRIu32 ", txn ID: %" PRIu64 ", pinned ID: %" PRIu64
      ", metadata pinned ID: %" PRIu64 ", name: %s",
      txn_session->id, __wt_atomic_loadv64(&txn_shared->id),
      __wt_atomic_loadv64(&txn_shared->pinned_id),
      __wt_atomic_loadv64(&txn_shared->metadata_pinned),
      txn_session->name == NULL ? "EMPTY" : txn_session->name));
    }
    
    if (!F_ISSET(txn, WT_TXN_HAS_SNAPSHOT))
          return (0);
```